docs/configuration/modules/dynawo-default-parameters.md

```diff
@@ -0,0 +1,53 @@
+---
+title: dynawo-default-parameters
+layout: default
+--- 
+
+The `dynawo-default-parameters` module defines the default values for all specific parameters of a dynamic simulation run with Dynawo. 
+
+# Properties
+
+## Required properties
+
+## parametersFile
+The `parametersFile` property is a required property that defines the path of the main parameters file.
+
+## solver.parametersFile
+The `solver.parametersFile` property is a required property that defines the path of the solver parameters file
+
+## Optional properties
+
+## network.parametersId
+The `network.parametersId` property is an optional property that defines the set of network parameters. The default value for this property is `NETWORK`.
+
```

Comment:
Here network.parametersFile should be added

Reply:
Done